openedx_tagging/core/tagging/fixtures/language_taxonomy.yaml

```diff
@@ -0,0 +1,1288 @@
+    value: Assamese
+    external_id: as
+- model: oel_tagging.tag
+  pk: 10
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Avaric
+    external_id: av
+- model: oel_tagging.tag
+  pk: 11
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Aymara
+    external_id: ay
+- model: oel_tagging.tag
+  pk: 12
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Azerbaijani
+    external_id: az
+- model: oel_tagging.tag
+  pk: 13
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Bashkir
+    external_id: ba
+- model: oel_tagging.tag
+  pk: 14
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Belarusian
+    external_id: be
+- model: oel_tagging.tag
+  pk: 15
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Bulgarian
+    external_id: bg
+- model: oel_tagging.tag
+  pk: 16
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Bihari languages
+    external_id: bh
+- model: oel_tagging.tag
+  pk: 17
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Bislama
+    external_id: bi
+- model: oel_tagging.tag
+  pk: 18
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Bambara
+    external_id: bm
+- model: oel_tagging.tag
+  pk: 19
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Bengali
+    external_id: bn
+- model: oel_tagging.tag
+  pk: 20
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Tibetan
+    external_id: bo
+- model: oel_tagging.tag
+  pk: 21
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Breton
+    external_id: br
+- model: oel_tagging.tag
+  pk: 22
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Bosnian
+    external_id: bs
+- model: oel_tagging.tag
+  pk: 23
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Catalan
+    external_id: ca
+- model: oel_tagging.tag
+  pk: 24
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Chechen
+    external_id: ce
+- model: oel_tagging.tag
+  pk: 25
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Chamorro
+    external_id: ch
+- model: oel_tagging.tag
+  pk: 26
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Corsican
+    external_id: co
+- model: oel_tagging.tag
+  pk: 27
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Cree
+    external_id: cr
+- model: oel_tagging.tag
+  pk: 28
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Czech
+    external_id: cs
+- model: oel_tagging.tag
+  pk: 29
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Church Slavic
+    external_id: cu
+- model: oel_tagging.tag
+  pk: 30
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Chuvash
+    external_id: cv
+- model: oel_tagging.tag
+  pk: 31
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Welsh
+    external_id: cy
+- model: oel_tagging.tag
+  pk: 32
+  fields:
+    taxonomy: 1
+    parent: null
+    value: Danish
+    external_id: da
+- model: oel_tagging.tag
+  pk: 33
+  fields:
+    taxonomy: 1
+    parent: null
+    value: German
```

Comment:
Please make sure we're thinking about how to translate these before displaying them in the UI. For a German user, this language tag needs to appear as "Deutsch", not "German".

We should probably use a similar mechanism for users, storing their user ID as the tag value but displaying their username or their "full name (username)" in the UI.

You don't have to implement this in this PR (it's a bad idea to, actually) - just think about how it will work in the future.

Reply:
> We should probably use a similar mechanism for users, storing their user ID as the tag value but displaying their username or their "full name (username)" in the UI.

What do you think of [this idea](https://github.com/open-craft/openedx-learning/pull/3#discussion_r1262170302)?

I think it could be adapted for translations too..

Reply:
I think something like that would make sense, yes. Just not clear on how it would be adapted to the current approach. But let's not worry too much about that just yet; I don't want to muddy the waters any further here :)